extensions/wikia/AdEngine/package-lock.json

```diff
@@ -5,8 +5,8 @@
   "requires": true,
   "dependencies": {
     "@wikia/ad-engine": {
-      "version": "git+https://github.com/Wikia/ad-engine.git#b7abf16884a430ea52e459d6c597ef1b667e1a66",
-      "from": "git+https://github.com/Wikia/ad-engine.git#v19.4.2",
+      "version": "git+https://github.com/Wikia/ad-engine.git#2ee56a834392cca420d9a7110eef512d4a8cc427",
```

Comment:
If you pushed new commits to this ad-engine branch, make sure you will rerun after-release task or update commit hash here. Otherwise it will install wrong version on sandbox during testing.